WebJobs.Script.sln

```diff
@@ -361,6 +361,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ManualTrigger-CSharp", "Man
 		sample\ManualTrigger-CSharp\run.csx = sample\ManualTrigger-CSharp\run.csx
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-Scrape-Promise", "HttpTrigger-Scrape-Promise", "{8E4F0ED5-75E0-4D0C-9469-05F738B1EC38}"
+	ProjectSection(SolutionItems) = preProject
+		sample\HttpTrigger-Scrape-Promise\function.json = sample\HttpTrigger-Scrape-Promise\function.json
```

Comment:
Probably name it something that includes `-Node` -- like `HttpTrigger-Promise-Node`.